main.js

```diff
@@ -43,6 +44,9 @@ for (let i = idx; i < process.argv.length; i++) {
     app.commandLine.appendSwitch(s, v);
 }
 
+// App Cache Path
+app.setPath('userData', path.join(process.env.APPDATA || (process.platform == 'darwin' ? process.env.HOME + 'Library/Preferences' : process.env.HOME + "/.local/share"), process.argv[4] || "astilectron"));
```

Comment:
I like using `process.argv[4]` to set the cache path.

However I'd rather have its value fully determined on GO side and simplifying here to 

```javascript
app.setPath('userData', process.argv[4]);
```